plugins/package-managers/go/src/main/kotlin/utils/GoSupport.kt

```diff
@@ -24,4 +24,4 @@ package org.ossreviewtoolkit.plugins.packagemanagers.go.utils
  * also any suffix starting with '+', because build metadata is not involved in version comparison according to
  * https://go.dev/ref/mod#incompatible-versions.
  */
-fun normalizeModuleVersion(moduleVersion: String): String = moduleVersion.removePrefix("v").substringBefore("+")
+internal fun normalizeModuleVersion(moduleVersion: String): String = moduleVersion.removePrefix("v").substringBefore("+")
```

Comment:
IMO the consistency argument (as written) has vanished a bit, since `escapeVersion()` no more is in this file. 
In my view, reducing visibility does not necessarily be done in this PR. 

However, the commit message should adhere to conventional commits.

I propose to just say

```
refactor(go)!: Reduce the visibility of `normalizeModuleVersion()`

There is no need to expose this function.
```

Reply:
Yes. Removed the change, so this way sticking to just **one** commit in the PR.